plugins/modules/vmware_vm_storage_policy.py

```diff
@@ -49,7 +49,7 @@
     - This parameter is ignored when C(state=absent).
     - This parameter is required when C(state=present).
     required: False
-    type: str
+    type: list
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.